src/pages/index.js

```diff
@@ -31,15 +31,19 @@ const P = styled.p`
       : ""}
 `;
 
-const Index = () => {
+const Index = ({ userCount }) => {
   return (
     <Layout title="Welcome" homepage>
       <h1>
         <span>Welcome to</span>
         Reactiflux
       </h1>
       <p>
-        We’re a chat community of 170,000+ React&nbsp;JS&nbsp;
+        We’re a chat community of{" "}
+        {`${new Intl.NumberFormat("en-US").format(
+          Math.floor(userCount / 1000) * 1000,
+        )}+`}{" "}
```

Comment:
I think it's better to use the default locale with the exact user count
```suggestion
        {new Intl.NumberFormat().format(
          userCount,
        )}{" "}
```

Reply:
I'd stick with the `en-US` locale as it correctly formats separators and decimals. Since all the copy is in hardcoded english with US spelling I'd not want the usercount to be formatted according to a different locale (German for example switches commas and colons). 

I can for sure use the exact count, I tried to stay in line with the hardcoded value which was an estimate rounded down to thousands.

Reply:
Good point, we can keep the locale consistent
```suggestion
        {new Intl.NumberFormat("en-US").format(userCount)}{" "}
```